google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -114,6 +114,23 @@
  *
  * BigtableDataClient client = BigtableDataClient.create(settings.build());
  * }</pre>
+ *
```

Comment:
I think this should be tied more to the paragraph of "The surface of this class includes several types of Java methods"

Reply:
Thanks for the pointer, Have updated accordingly.